cumulusci/cli/cci.py

```diff
@@ -1419,3 +1396,75 @@ def flow_run(runtime, flow_name, org, delete_org, debug, o, skip, no_prompt):
                 "Scratch org deletion failed.  Ignoring the error below to complete the flow:"
             )
             click.echo(str(e))
+
+
+@error.command(name="info", help="Outputs the last part of the most recent traceback")
+@click.argument("num_lines", required=False)
+def info(num_lines):
+    if not CCI_LOGFILE_PATH.is_file():
+        click.echo(f"No logfile found at: {CCI_LOGFILE_PATH}")
+        return
+
+    if not num_lines:
+        output_lines = 30
```

Comment:
If we switch to an option we can define it like:

```python
@click.option('--num_lines', '-n', default=30, show_default=True, type=int)
```

...and won't need to do it here.